tf-modules/data-persistence/terraform.tfvars.example

```diff
@@ -19,4 +19,4 @@ enable_point_in_time_tables = [
 ]
 include_elasticsearch  = true
 subnet_ids             = ["subnet-123456"]
-security_groups        = ["sg-123456"]
```

Comment:
While you're at it you may want to delete `create_service_linked_role` from, and add `custom_domain_name` to, this example file.